low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -80,6 +82,11 @@ class NodeDataMsgHandler(ScheduledModuleThread, InternalMsgQ):
         'disk' : False,
         'memory' : False
     }
+    check_time = {

```

Comment:
Please choose a meaningful name for this variable.

Reply:
How about `curr_threshold_timestamp` ?

Reply:
Done